foundations/src/telemetry/tracing/live/mod.rs

```diff
@@ -1,34 +1,108 @@
 mod event_output;
 mod live_reference_set;
 
-use cf_rustracing_jaeger::span::Span;
-use live_reference_set::{LiveReferenceHandle, LiveReferenceSet};
+use cf_rustracing_jaeger::Span;
+use live_reference_set::LiveReferenceSet;
+use parking_lot::RwLock;
 use std::sync::Arc;
 use std::time::SystemTime;
 
-type SharedSpanInner = Arc<parking_lot::RwLock<Span>>;
-pub(crate) type SharedSpanHandle = Arc<LiveReferenceHandle<SharedSpanInner>>;
+pub(crate) use live_reference_set::LiveReferenceHandle;
+
+use crate::telemetry::settings::LivenessTrackingSettings;
+use crate::telemetry::tracing::internal::SharedSpanHandle;
```

Comment:
Nit: here and in other places: don't separate those `use`'s from the rest of the non-public imports